### PR TITLE
Lfs support plus tests nits

### PR DESCRIPTION
**Pull request by ricarkol**
3 changes:
- adding support for LFS disks: adding the lfs library, and mount(LFS) to rumprun setup.
- blk_test is more useful now: it gets a path as an arg and just prints the file content if file, or sub-files if directory.
- rootfs_mount_test now checks that /dev/urandom is present in the unikernel FS.

### CODE REVIEW

File: blk_test/Makefile [hunk 1]
```diff
@@ -43,8 +43,15 @@ run_ukvm: blk-rumprun.ukvm test.iso test.ext2
...
+	../ukvm-bin.seccomp --disk=test.iso --net=tap100 blk-rumprun.seccomp '{"cmdline":"blk /test/aaaaaaaaaaaaaaax","blk":{"source":"etfs","path":"/dev/ld0a","fstype":"blk","mountpoint":"/test"}}'
+	../ukvm-bin.seccomp --disk=test.iso --net=tap100 blk-rumprun.seccomp '{"cmdline":"blk /test", "blk":{"source":"etfs","path":"/dev/ld0a","fstype":"blk","mountpoint":"/test"}}'
+	e2tail test.ext2:/bla
+
+.PHONY: run_seccomp_lfs
+run_seccomp_lfs: blk-rumprun.seccomp test.lfs
+	touch dummy
+	../ukvm-bin.seccomp --disk=test.lfs --net=tap100 blk-rumprun.seccomp '{"cmdline":"blk /test/aaaaaaaaaaaaaaax","blk":{"source":"etfs","path":"/dev/ld0a","fstype":"blk","mountpoint":"/test"}}'
+	../ukvm-bin.seccomp --disk=test.lfs --net=tap100 blk-rumprun.seccomp '{"cmdline":"blk /test", "blk":{"source":"etfs","path":"/dev/ld0a","fstype":"blk","mountpoint":"/test"}}'
 	e2tail test.ext2:/bla
```
Comment (sahilsuneja1) [inline blk_test/Makefile]: whats happening here? should this be test.lfs?

**Review by sahilsuneja1 — APPROVED** [review]
LGTM

File: blk_test/Makefile (hunk 1 above)
Comment (ricarkol) [reply blk_test/Makefile]: ah, that shouldn't be there. thanks

Comment (ricarkol) [reply blk_test/Makefile]: removed with the latest commit (which was squashed)

**Pull request merged**